src/vanilla/DanModel/JavaBlock.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+
+namespace AutoRest.Java.DanModel
+{
+    public class JavaBlock
```

Comment:
Is this like a facade to convey to readers of a large fluent chain what construct we're in right now?

Reply:
Yes, and it also serves as a way of restricting what you can add to the chain within a block. For example, there's no `Import()` method on JavaBlock, because you can't have an `import` statement there. That method only exists on JavaFile.